app/src/components/app-nav/types.ts

```
@@ -0,0 +1,5 @@
+import { AppLogo } from '../app-logo/app-logo.tsx'
+import { NavMenu } from '../nav-menu/nav-menu.tsx'
+
+export type AppLogo = typeof AppLogo
```

Biome (2.3.13)

[error] Shouldn't redeclare 'AppLogo'. Consider to delete it or rename it.

'AppLogo' is defined here:


(lint/suspicious/noRedeclare)